storages/multi-threaded-memory-storage/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "gluesql-multi-threaded-memory-storage"
```

Comment:
How about shorter name?
e.g. `gluesql-shared-memory-storage`
This storage itself is single threaded but it can be accessed by multiple different threads.
The term shared memory implies that memory data can be accessed my multiple process, so it would be good description for this.

Reply:
I agree with the use of short names.

Can you explain a bit more what you mean by single-threaded storage?

Reply:
Good point, "single-threaded storage" keyword may be not good to describe it.
The key difference between this and `MemoryStorage` is whether the storage can be accessed concurrently or not by default.
I thought `MultiThreaded-` meaning as a kind of that the storage itself spawns multiple threads to handle data faster.

Reply:
> Good point, "single-threaded storage" keyword may be not good to describe it. The key difference between this and `MemoryStorage` is whether the storage can be accessed concurrently or not by default. I thought `MultiThreaded-` meaning as a kind of that the storage itself spawns multiple threads to handle data faster.

Oh, I understand!
thank you